tasks.py

```diff
@@ -324,3 +324,10 @@ def _replace(pattern):
     run("git commit -a -m 'Automated update for release v{}'".format(version), echo=True)
     run("git tag v{} -m 'See the release notes for details:\n\nhttps://metallb.universe.tf/release-notes/#version-{}-{}-{}'".format(version, version.major, version.minor, version.patch), echo=True)
     run("git checkout main", echo=True)
+
```

Comment:
We have only one empty line between functions in other places.

Reply:
All of the other places are wrong, lol.

https://www.python.org/dev/peps/pep-0008/#blank-lines

... I could do a follow-up to fix up formatting and enforce it with a CI job.